lib/scala/tree/src/main/scala/io/github/tetherlessworld/twxplore/lib/tree/TreeCsvTransformerSink.scala

```diff
@@ -0,0 +1,94 @@
+package io.github.tetherlessworld.twxplore.lib.tree
+
+import java.io.{BufferedWriter, File, FileWriter}
+
+import io.github.tetherlessworld.scena.Rdf
+import io.github.tetherlessworld.twxplore.lib.geo.models.domain._
+import org.apache.jena.rdf.model.{Model, ModelFactory}
+
+trait TreeCsvTransformerSink {
```

Comment:
The trait passed to parse should be an interface, no implementation. Create an implementation that does the model-per-file, have EtlCommand new this implementation and pass it in.